parser/src/lib.rs

```diff
@@ -10,13 +10,15 @@
 use std::borrow::Cow;
 use std::ops::Range;
 
-/// A `{{:parameter}}` in a format string.
+/// A `{{$index=$type:$hint}}` in a format string.
```

Comment:
nit: it's hard to figure out whether the $ indicates a variable or is part of the syntax